src/api/ResponseHelper.js

```diff
@@ -38,11 +38,11 @@ const parseErrorArray = (errors) =>
  * @param {Object} data e.g. { data: { id: 1}}
  * @returns { Boolean, {Object}, String} { success, data, errors } e.g. { success: true, data: {id: 1}} or {success: false, errors: 'there was an error'}
  */
-const newResponse = ({ success, data, error }) => ({
+const newResponse = ({ success, data, error, errors }) => ({
```

Comment:
Why does this handle both a singular error and the plural errors?  I feel there's an inefficiency somewhere in that.

Reply:
I suspect they are different kinds of errors so not sure it is an efficiency situation. More likely the errors need renaming. It might be that error is the error returned from axios e.g. network error and errors are probably returned from the service. They have a different structures. I will add a note to it to come back to later.